EIPS/eip-2400.md

```diff
@@ -14,14 +14,21 @@ requires: 831
 
 A standardized URI for transaction receipt with complete information about transaction.
 
-A standard way of representing a submitted transaction.
+A standard way of representing a transaction receipt.
```

Comment:
not sure about this change - under a "transaction receipt" I understand something different. I thought it was better before the change

Reply:
Not sure as well, the terminology for transaction receipt implies that the transaction was submitted, as the URI does not contain the transaction itself, just the hash of it, and any methods a events attached to it would need to be verified by the receiver/reader.  

Reading a transaction URI of a transaction not publicized would render an error until the transaction can be found.

A pending transaction can be displayed if found, but it cannot be guaranteed (as it could be overrun, fail or end up in a fork in unusual scenarios).

Reply:
But I see the wording before the change can be more "understandable", and as the above paragraph already mention of transaction receipt, I guess I would leave it as it was.